src/standalone/Vegetation/autotrophic_respiration.jl

```diff
@@ -92,13 +90,14 @@ function compute_autrophic_respiration(
     h,
 )
 
-    (; ne, ηsl, σl, μr, μs, f1, f2) = model.parameters
+    (; ne, ηsl, σl, μr, μs, f) = model.parameters
     Nl, Nr, Ns =
         nitrogen_content(ne, Vcmax25, LAI, SAI, RAI, ηsl, h, σl, μr, μs)
-    Rpm = plant_respiration_maintenance(Rd, β, Nl, Nr, Ns, f1)
-    Rg = plant_respiration_growth(f2, An, Rpm)
+    Rpm = plant_respiration_maintenance(Rd, β, Nl, Nr, Ns)
```

Comment:
`Rpm = Rd * (β + (Nr + Ns) / max(Nl, eps(FT)))` uses mol CO2
and 
` Rg = f * (An - Rpm)` here `f` does not do a conversion, so this is still in mol CO2

right?

Reply:
exact!